tests/test/org/commcare/backend/session/test/BasicSessionNavigationTests.java

```diff
@@ -129,4 +129,45 @@ public void testStepBackWithComputedDatum() {
         Assert.assertEquals(session.getNeededData(), SessionFrame.STATE_COMMAND_ID);
     }
 
+    @Test
+    public void testStepToSyncRequest() {
+        SessionWrapper session = mApp.getSession();
+
+        session.setCommand("patient-search");
+        Assert.assertEquals(session.getNeededData(), SessionFrame.STATE_QUERY_REQUEST);
+        session.setQueryDatum(null);
+
+        /*
```

Comment:
oops, it is hold over from before I introduced the `RemoteQuerySessionManager`. I'll remove it